completions/ssh

```diff
@@ -434,8 +434,11 @@ _comp_cmd_scp__path_esc='[][(){}<>"'"'"',:;^&!$=?`\\|[:space:]]'
 # Complete remote files with ssh.  If the first arg is -d, complete on dirs
 # only.  Returns paths escaped with three backslashes.
 # shellcheck disable=SC2120
+# @param  $1 `no-overescape` if we should use 1 backslash for escaping instead of 3
 _comp_xfunc_ssh_scp_remote_files()
 {
+    local no_overescape=$1
```

Comment:
Actually, with the current approach, the `set -o nounset` (`set -u`) should be avoided in this line.